Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -2544,8 +2588,9 @@ GC::Ref<ComputedProperties> StyleComputer::compute_properties(DOM::Element& elem
     // 6. Run automatic box type transformations
     transform_box_type_if_needed(computed_style, element, pseudo_element);
 
-    // 7. Resolve effective overflow values
+    // 7. Apply any property-specific computed value logic
```

Comment:
I realised that the missing "step where we produce computed values" is actually right here, it just had an overflow-specific comment. :laughing: I'll try and get the others sorted out soon.